prs/PizzaStore.ts

```diff
@@ -0,0 +1,40 @@
+class PizzaStore {
+    public createPizza(style: string, type: string): Pizza | null {
+        let pizza: Pizza = undefined;
+
+        if (style === "NY") {
```

Comment:
Consider replacing all these hard-coded strings with typescript enums to reduce the potentials for errors or misuse.